emsdk.py

```diff
@@ -81,6 +78,20 @@
 
 UNIX = (OSX or LINUX)
 
+
+# Pick which shell of 4 shells to use
+POWERSHELL = bool(os.getenv('EMSDK_POWERSHELL'))
+CSH = bool(os.getenv('EMSDK_CSH'))
+BASH = bool(os.getenv('EMSDK_BASH'))
+CMD = bool(os.getenv('EMSDK_CMD'))
+if not CSH and not POWERSHELL and not BASH and not CMD:
+  # Fall back to default of `cmd` on windows and `bash` otherwise
+  if WINDOWS and not MSYS:
+    CMD = True
+  else:
+    BASH = True
```

Comment:
this change looks unrelated to testing, so the PR is doing more than that? Or is this NFC somehow?

Reply:
Sorry that is a separate change that I'm landing first.